students/FyfyNguyen/session05/exceptions_lab.py

```diff
@@ -0,0 +1,17 @@
+# !/usr/bin/python3
+
+
+def safe_input(response):
+    """
+    Return None if user enters ^C or ^D instead of raising
+    exceptions EOFError or KeyboardInterrupt.
+    """
+    try:
+        return input(response)
+    except EOFError:
```

Comment:
you can also do:

```
except (EOFError, KeyboardInterrupt):
    ...
```
the other form is for when you do something different with the different exceptions.